statistics-dashboard-common/src/main/java/eu/europeana/statistics/dashboard/common/iternal/FacetValue.java

```diff
@@ -0,0 +1,29 @@
+package eu.europeana.statistics.dashboard.common.iternal;
+
+import com.fasterxml.jackson.annotation.JsonValue;
+
+public enum FacetValue {
```

Comment:
This is good, but I think we could make it better. There are now three places where these names are defined separately:
1. This class `FacetValue` (as the enum name), 
2. The class `FilteringOptions` (as the `@JsonProperty` name),
3. The class `StatisticsFilteringRequest` (as the `@JsonProperty` name).

The UI requires these names to be identical. So perhaps we could create a list of constants somewhere that is used in all three of these places?

Reply:
Just created a new class